dockers/actions-plan-preview/main.go

```diff
@@ -48,9 +50,10 @@ func main() {
 		&oauth2.Token{AccessToken: args.Token},
 	)
 	tc := oauth2.NewClient(ctx, ts)
-	ghClient := github.NewClient(tc)
+	ghClientV3 := github.NewClient(tc)
+	ghClientV4 := githubv4.NewClient(tc)
```

Comment:
How about keeping the old one as it is and using `ghGraphQLClient` for the new one?